docs/setup.asciidoc

```diff
@@ -5,6 +5,7 @@ All you need is:
 
 * Elasticsearch {esversion}
 * A modern web browser - http://www.elastic.co/subscriptions/matrix#matrix_browsers[Supported Browsers].
+* 512 MB of available RAM
```

Comment:
Can you add a note about `--max-old-space-size` and mention that it is set to 256 by default, but that users can expect Kibana to use a bit more? Doesn't need to be added here, but somewhere in the docs